### PR TITLE
Utbedring etter tilbakemeldinger

### PR DESCRIPTION
**Pull request by ViktorGSolberg**
Basert på [dette kortet](https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-10084)

* https://github.com/navikt/familie-klage/pull/142

### CODE REVIEW

File: src/frontend/App/context/BehandlingContext.tsx [hunk 1]
```diff
@@ -40,10 +43,11 @@ const [BehandlingProvider, useBehandling] = constate(() => {
         hentVilkårsvurderinger(behandlingId);
     }, [behandling, behandlingId, hentVilkårsvurderinger]);
     useEffect(() => {
-        settVilkårOppfyltOgBesvart(
+        settFormkravOppfylt(
             vilkårsvurderinger.status === RessursStatus.SUKSESS &&
+                påKlagetVedtakValgt(vilkårsvurderinger.data) &&
                 alleVilkårOppfylt(vilkårsvurderinger.data) &&
-                harVerdi(vilkårsvurderinger.data.saksbehandlerBegrunnelse)
+                begrunnelseUtfylt(vilkårsvurderinger.data)
```
Comment (charliemidtlyng) [inline src/frontend/App/context/BehandlingContext.tsx:50]: Denne gir ikke mening ift oppfylt. Da skal ikke begrunnelse være med

File: src/frontend/Komponenter/Behandling/Formkrav/EndreFormkravVurderinger.tsx [hunk 2]
```diff
@@ -42,37 +50,50 @@ const HelpTextContainer = styled.div`
...
+
     const submitOppdaterteVurderinger = () => {
         if (oppdatererVurderinger) {
             return;
         }
         settOppdatererVurderinger(true);
 
-        lagreVurderinger(vurderinger).then((res: Ressurs<IFormkravVilkår>) => {
+        const vurderingerSomSkalLagres = alleVilkårErOppfylt
+            ? vurderinger
```
Comment (charliemidtlyng) [inline src/frontend/Komponenter/Behandling/Formkrav/EndreFormkravVurderinger.tsx:90]: motsatt?

File: src/frontend/Komponenter/Behandling/Formkrav/KlageInfo.tsx [hunk 3]
```diff
@@ -49,84 +39,19 @@ const TabellRad = styled.div`
...
     const utledetIkon = () => {
         if (redigeringsmodus === Redigeringsmodus.IKKE_PÅSTARTET) {
             return <Advarsel height={26} width={26} />;
-        } else if (
-            alleVilkårOppfylt(vurderinger) &&
-            harVerdi(vurderinger.saksbehandlerBegrunnelse)
-        ) {
-            return <OppfyltIkonStyled height={23} width={23} />;
-        } else if (alleVilkårOppfylt(vurderinger)) {
+        } else if (formkravOppfylt) {
```
Comment (charliemidtlyng) [inline src/frontend/Komponenter/Behandling/Formkrav/KlageInfo.tsx:52]: Må ikke påklagetVedtakValgt ligge i if-en her?

**Review by charliemidtlyng — APPROVED** [review]
Parprogget endringer sammen

File: src/frontend/Komponenter/Behandling/Formkrav/KlageInfo.tsx (hunk 3 above)
Comment (ViktorGSolberg) [reply src/frontend/Komponenter/Behandling/Formkrav/KlageInfo.tsx:52]: [265b015](https://github.com/navikt/familie-klage-frontend/pull/156/commits/265b01541099b944e9524b11c08ba13565798f71)

File: src/frontend/Komponenter/Behandling/Formkrav/EndreFormkravVurderinger.tsx (hunk 2 above)
Comment (ViktorGSolberg) [reply src/frontend/Komponenter/Behandling/Formkrav/EndreFormkravVurderinger.tsx]: [265b015](https://github.com/navikt/familie-klage-frontend/pull/156/commits/265b01541099b944e9524b11c08ba13565798f71)

File: src/frontend/App/context/BehandlingContext.tsx (hunk 1 above)
Comment (ViktorGSolberg) [reply src/frontend/App/context/BehandlingContext.tsx]: [265b015](https://github.com/navikt/familie-klage-frontend/pull/156/commits/265b01541099b944e9524b11c08ba13565798f71)

**Pull request merged**